source/includes/_comparison.md

```diff
@@ -1,26 +1,29 @@
 ## Comparison
-Retrieves both `Readings` and the `interval`-th `Predictions` in the last `horizon` span specified.
+Retrieves both `Readings` and `Predictions` for an user specified period(`horizon`). Users are also able to select the set of `Predictions` based on the `interval` specified.
 
 ### Usage
 
           | Value
 ---------:|:-----
 __endpoint__ | `https://api.metislabs.tech/1.0/comparison`
 __method(s)__ | `GET`
-__role__ | `supervisor`
 
 #### Parameters
 
 The `Default` column specifies the value used when the user does not specify a parameter in the request.
 
 Field | Type | Default | Limitation | Description
 -----:|:----:|:---------:|:-----:|:-----------
-__horizon__ | float | NA | x > 0 | A float value which represents the number of hours of `Readings` the user specifies in the request.
-__interval__ | int | NA | x in (5, 10, 15) | An integer value which represents the set of `Predictions` to be retrieved, has to be either 5, 10 or 15.
+__organisation__ | string | NA | NA | The organisation that the user belongs to.
+__site__ | string | First site available to the user | NA | The site that the user wants to retrieve data from.
+__process__ | string | None  | NA | The process that the user wants to retrieve data from.
+__forecast__ | stirng | None | NA | The forecast that the user wants to retrieve data from.
+__horizon__ | float | 2 | x > 0 | A float value which represents the number of hours of `Readings` the user specifies in the request.
+__interval__ | int | 5 | x in (5, 10, 15) | An integer value which represents the set of `Predictions` to be retrieved, has to be either 5, 10 or 15.
 
 * Each [Prediction](#predictions) is classed by the intervals it was made in.
 * Currently only 5, 10 and 15 minute `Predictions` are available upon request.
-* If the `horizon` and `interval` parameters are not specified or incorrectly specified, a [URLParameterError](#client-based-errors) will be returned in the response.
+* If the `horizon` and `interval` parameters are not specified or incorrectly specified, a [URLParameterError](#client-based-exceptions) will be returned in the response.
```

Comment:
Only if 'incorrectly specified' now, is they're not specified we get the default